Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,53 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        incident = {}
+
+        incident_csvs = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []

```

Comment:
Is **incident_csvs** mean multiple csv files that are attached to the alert, or is it the actual records inside one csv attachment?

Reply:
**incident_csvs** mean actual records inside one csv attachment. I will change the name to `incident_csv_records`